import-export-cli/integration/testutils/api_testUtils.go

```diff
@@ -176,6 +177,7 @@ func ValidateAllApisOfATenantIsExported(t *testing.T, args *ApiImportExportTestA
 		//Remove Exported apis and logout
 		pathToCleanUp := utils.DefaultExportDirPath + utils.TestMigrationDirectorySuffix
 		base.RemoveDir(pathToCleanUp)
+		base.WaitForIndexing()
```

Comment:
No need to wait here because we are only removing the export archive file and nothing related to indexing is taking place in APIM side.